### DISCUSSION TITLE
What are these patches licensed under?

### DISCUSSION THREAD

**Discussion by nonetrix** [Frogging-Family/linux-tkg#887] [Q&A] [answered]
I don't see any license file, any clarification? I would assume GPL V2, although I am not a legal expert

**Comment by nonetrix** [comment]
Related
https://github.com/NixOS/nixpkgs/issues/287189

**Comment by AdelKS** [comment]
I believe most, if not all, patches in this repo are taken from elsewhere, so we don't have a say on their license. Let's look into each patch individually and see, because there are commit messages in them usually.

If you are asking about the original scripts within this repo, yeah that isn't clear.

Also, note that `linux-tkg` isn't a single kernel, it's a build system that can produce a lot of slightly different kernels. The biggest difference although would be the scheduler, look at the releases page and you'll see that we provide 4 flavors of tkg kernels already.

**Comment by Tk-Glitch** [comment, marked as answer]
Outside of Project C which is GPL 3.0, all other patches are GPL 2.0.
Regarding our build scripts, this falls into public domain atm. Please let me know if you need anything regarding packaging for NixOS and I'll happily help.

**Comment by nonetrix** [comment]
Safe to close I think
